deploy/olm/operator_community.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: test-csm-operator-olm
 spec:
   sourceType: grpc
-  image: quay.io/dell/container-storage-modules/dell-csm-operator:v1.8.1
+  image: <replace with catalog image generated by  'make catalog-build-fbc' >
 ---
```

Comment:
Was this just a cleanup change?

Reply:
that image should be an internal image generated by the user testing out OLM. 
I wanted to remove the operator image there since that's incorrect and will result in a lot of wasted time on debugging.